site/README.md

```diff
@@ -49,10 +49,14 @@ To see changes in Javadoc, run:
 
 ### Publishing
 
-After site changes are committed, you can publish the site with this command:
+After site changes are committed, you can publish them by running the following sequence of commands from the `$projectRoot/site` directory:
```

Comment:
I think `$projectRoot` makes this more confusing than necessary. I think this should just refer to the `site` directory. If you want to make it more clear than that, then state that it's the directory where this README.md is located:

> ... sequence of commands from the `site` directory (where this `README.md` is located):